src/http-schema/http-schema.ts

```diff
@@ -75,12 +75,12 @@ const ImageObjectContent = Joi.object({
 
 // These are not imported yet
 const OtherContent = Joi.object({
-  type: Joi.string().valid('CodeBlock', 'MathFragment', 'MediaObject', 'Table', 'ThematicBreak'),
+  type: Joi.string().valid('CodeBlock', 'MathFragment', 'MediaObject', 'Table', 'ThematicBreak', 'List'),
```

Comment:
We support lists in the client so this should be added to the validator rather than to the list of `OtherContent`